nexus/src/app/silo.rs

```diff
@@ -225,6 +226,36 @@ impl super::Nexus {
         Ok(shared::Policy { role_assignments })
     }
 
+    pub(crate) async fn silo_fetch_settings(
+        &self,
+        opctx: &OpContext,
+        silo_lookup: &lookup::Silo<'_>,
+    ) -> LookupResult<SiloAuthSettings> {
+        // TODO: can everyone view this on their own silo? why not, right?
```

Comment:
hmm, i can kind of imagine not wanting to show everyone the silo's authz settings, IDK...